src/Microsoft.ServiceHub.Framework/ServiceHub/LazyAuthorizationServiceProxy.cs

```diff
@@ -19,9 +19,8 @@ internal class LazyAuthorizationServiceProxy : IAuthorizationService, IDisposabl
 	/// </summary>
 	/// <param name="serviceBroker">A service broker used to acquire the activation service.</param>
```

Comment:
This parameter doc should be updated to indicate that it is optional, and describe the behavior of the class when the argument is `null`.